edith-step/src/ReceiptInput.test.js

```diff
@@ -4,52 +4,74 @@ import { shallow, configure, mount } from 'enzyme';
 import ReceiptInput from './ReceiptInput';
```

Comment:
can we removed the unused imports ? 
render, shallow, configure seem to be not used